meson_options.txt

```diff
@@ -1,5 +1,6 @@
 option('daemon', type : 'boolean', value : true, description : 'enable the fwupd daemon')
 option('consolekit', type : 'boolean', value : true, description : 'enable ConsoleKit support')
+option('contrib', type : 'boolean', value : true, description : 'enable contrib installation')
```

Comment:
I'd much rather `firmware_packager` here -- on the logic we don't install everything in contrib now.

Reply:
Actually I proposed contrib to @djcampello on purpose, with the idea that this scales better to not install anything in contrib as we add more.

Reply:
...but contrib isn't really a "thing" it's just a place we put stuff that doesn't belong anywhere else...

Reply:
Well I guess that's true.  We should probably then adopt a policy that as new stuff shows up in contrib that "could" install it needs to grow an option like this.